evm/src/main/java/org/hyperledger/besu/evm/internal/EvmConfiguration.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright contributors to Hyperledger Besu.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.hyperledger.besu.evm.internal;
+
+public class EvmConfiguration {
```

Comment:
have you decided to not use the Immutable library for this object ?

Reply:
Not specifically. I'll add that in a later pr.